components/Card/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import styled from "styled-components";
+
+interface Props extends React.Attributes {
+  onClick?: () => void | undefined;
```

Comment:
```suggestion
  onClick?: React.MouseEventHandler<HTMLElement>;
```

btw, `void` and `undefined` have more or less the same meaning.